src/runtime/virtcontainers/virtiofsd_test.go

```diff
@@ -130,12 +130,7 @@ func TestValid(t *testing.T) {
 		{"source is not available", func(v *virtiofsd) {
 			v.sourcePath = "/foo/bar"
 		}, errVirtiofsdSourceNotAvailable, nil},
-		{"replace cache mode none by never", func(v *virtiofsd) {
-			v.cache = "none"
-		}, nil, func(name string, a *assert.Assertions, v *virtiofsd) {
-			a.Equal("never", v.cache, "test case %+s, cache mode none should be replaced by never", name)
-		}},
-		{"invald cache mode: replace none by never", func(v *virtiofsd) {
+		{"invald cache mode", func(v *virtiofsd) {
```

Comment:
`invald` -> `invalid` (which I know was there before, but worth fixing in this.

Reply:
Good catch. Fixed. Thanks @stevenhorsman !